x-pack/plugins/actions/server/builtin_action_types/servicenow/utils.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Incident, PartialIncident, ResponseError } from './types';
+import { FIELD_PREFIX } from './config';
+import { addTimeZoneToDate, getErrorMessage } from '../lib/axios_utils';
+import * as i18n from './translations';
+
+export const prepareIncident = (useOldApi: boolean, incident: PartialIncident): PartialIncident =>
+  useOldApi
+    ? incident
+    : Object.entries(incident).reduce(
+        (acc, [key, value]) => ({ ...acc, [`${FIELD_PREFIX}${key}`]: value }),
+        {} as Incident
+      );
+
+const createErrorMessage = (errorResponse: ResponseError): string => {
+  if (errorResponse == null) {
+    return 'unknown';
+  }
+
+  const { error } = errorResponse;
+  return error != null ? `${error?.message}: ${error?.detail}` : 'unknown';
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const createServiceError = (error: any, message: string) =>
```

Comment:
No, but you are right, it can be fixed. You made me think of a solution 🙂. Thanks!

I can set it as `error: ResponseError` and extend `ResponseError` with `AxiosError`.